charter.html

```diff
@@ -234,28 +234,42 @@ <h2 id=proposals>Proposals</h2>
 
 <ul>
 <li>because the Proposal has become sufficiently complex that writing an
-explainer is warranted
+<a href=https://github.com/w3ctag/w3ctag.github.io/blob/master/explainers.md>explainer</a>
+is warranted
 <li>because the Proposal is sufficiently complex that having a dedicated
 issue tracker for it is warranted
 <li>because multiple <a href=#implementer>implementers</a> have
 expressed interest in the Proposal
 </ul>
 
+<p>Artifacts produced by successful Proposals may include an explainer
+describing the proposed changes to the web platform. Such an explainer
+may serve as the basis for a Work Item, should the Proposal be adopted
+as one. Proposals cannot result in
+<a href=https://www.w3.org/community/reports/>Community Group
+Reports</a>.
+
 <p>Proposals may be withdrawn by their originators, or may be closed by
 the Chairs (if, for example, the Chairs deem the Proposal to
 be <a href=out-of-scope>out of scope</a> or the Proposal fails to gain
 sufficient <a href=#implementer>implementer</a> support to be adopted as
 a Work Item). If such a Proposal has a dedicated repository, the Chairs
-should take steps to ensure the data is not lost, perhaps by transfering
-the repository to a different organization or user, or by archiving it.
+should take steps to ensure the data is not lost, perhaps by
+transferring the repository to a different organization or user, or by
+archiving it.
 
 <h2 id=work-items>Work Items</h2>
 
-<p>A <dfn>Work Item</dfn> is a Proposal that has been formally adopted
-as something the Community Group will work on. Every Work Item has one
-or more <a href=#editors>Editors</a>, who are appointed by
+<p>A <dfn>Work Item</dfn> is something that has been formally adopted
+for the Community Group to work on. Every Work Item has one or
+more <a href=#editors>Editors</a>, who are appointed by
 the <a href=#chairs>Chairs</a>.
 
+<p class=note>Work Items typically begin life as a Proposal before being
+formally adopted, and are expected to result in a
+<a href=https://www.w3.org/community/reports/>Community Group Report</a>
```

Comment:
Apparently not all Reports are Specifications so it might just be clearer to write

```suggestion
<a href=https://www.w3.org/community/reports/>Specification</a>
```

here.

Reply:
I think I've improved this in subsequent edits. Let me know if things are still unclear.